src/popup/handlers/launch-panel-header-click-handler.ts

```diff
@@ -14,7 +14,11 @@ export class LaunchPanelHeaderClickHandler {
         if (item == null) {
             return;
         }
+
         const url: string = item.data;
+
+        // the following warning is thrown incorrectly because the call to 'open' is not on the file system
```

Comment:
Would be good to link the bug tracking the false positive, https://github.com/nodesecurity/eslint-plugin-security/issues/54